<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">OpenMoldyn</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_model">
       <attribute name="title">
        <string>Model</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QToolButton" name="newModelBtn">
          <property name="text">
           <string>New</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="4">
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Current model</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QTreeWidget" name="paramsTreeWidget">
             <property name="indentation">
              <number>10</number>
             </property>
             <column>
              <property name="text">
               <string>Property</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Value</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Unit</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QToolButton" name="loadModelBtn">
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="4">
         <widget class="QCommandLinkButton" name="gotoSimuBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Simulation</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_simu">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>Simulation</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="3" column="0" colspan="2">
         <widget class="QGroupBox" name="progress_groupBox">
          <property name="title">
           <string>Progress</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QProgressBar" name="simuProgressBar">
             <property name="maximum">
              <number>1</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
             <property name="format">
              <string>%v/%m</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QToolButton" name="RTViewBtn">
             <property name="text">
              <string>View current state</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCommandLinkButton" name="simuBtn">
          <property name="text">
           <string>Launch simulation</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="iterationsLabel">
            <property name="text">
             <string>Iterations</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="iterationsSpinBox">
            <property name="maximum">
             <number>210000766</number>
            </property>
            <property name="singleStep">
             <number>100</number>
            </property>
            <property name="value">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="simulationTimeLabel">
            <property name="text">
             <string>Simulation time (s)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="simulationTimeLineEdit"/>
          </item>
         </layout>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QCommandLinkButton" name="gotoProcessBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Process data</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_processing">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>Processing</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>499</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New model</string>
   </property>
  </action>
  <action name="actionOpen_model">
   <property name="text">
    <string>Open model</string>
   </property>
  </action>
  <action name="actionNew_simulation">
   <property name="text">
    <string>New simulation</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
